Assets/script.js

```diff
@@ -320,13 +318,13 @@ function callWeatherApi() {
     $.ajax({
         url: queryUrl,
         method: "GET"
-            // process retrieved weather data
+        // process retrieved weather data
     }).
-    then(processHourlyWeatherData).
+        then(processHourlyWeatherData).
         // catch error if call is unsuccessful
-    catch(function(error) {
-        console.log("Catch error");
-    });
+        catch(function (error) {
+            console.log("Catch error");
+        });
```

Comment:
remove the whitespace by backtabbing these lines, remove whitespace between function and (error) to make function(error)